clippy_lints/src/utils/author.rs

```diff
@@ -97,7 +97,8 @@ impl<'a, 'tcx> LateLintPass<'a, 'tcx> for Author {
             return;
         }
         prelude();
-        PrintVisitor::new("var").visit_variant(var, &hir::Generics::empty(), hir::DUMMY_HIR_ID);
+        let dummy_hir_id = cx.tcx.hir().get_parent_node(var.id);
```

Comment:
This is not dummy. This is the correct `HirId` for the HIR visitor method.

Reply:
Eh.. :(
I'll change it in a follow up pr that we can add to the rollup

Reply:
GHA got stuck anyway in the caching process.